_projects/community-share.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Community Share
+date: 2015-06-01
+link: https://github.com/communityshare/communityshare
+intro: Come help with
+image: /assets/images/communityshare.png
+lead: Josh Schachter & Ben Reynwar
+contact: http://www.communityshare.us/contact/
```

Comment:
It would be nice to keep the html out of the *.md files, but I think its totally fine for now.